### PR TITLE
Add Console project for Tolower

### PR DESCRIPTION
**Pull request by cihanozluk**

### CODE REVIEW

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+                foreach (char item in Word)
+                {
+
+                    lowerWord.Append(GetToLowerChar(item, str));
+                }
+
+                if (Language == "el" && Word.Substring(Word.Length - 1, 1) == "Σ")
```
Comment (pnklshh) [inline S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:51]: Comparing the language strings will not satisfy other varieties of 'el'. For example 'el-AB', 'el-BC'(this is just an example)

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                {
+                    lowerWord[Word.Length - 1] = 'ς';
+                }
+                if ((Language == "ga" || Language == "ga-IE") && (Word.Substring(0, 1) == "n" || (Word.Substring(0, 1) == "t" && str.Upper.IndexOf(Word.Substring(1, 1)) >= 0)))
```
Comment (pnklshh) [inline S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:55]: 1. Comparing the language strings will not satisfy other varieties of 'ga'. For example 'ga-IA', 'ga-IB'(this is just an example)
2. Can you explain last condition with some comment ?

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            alphabets.Add(new Alphabet("el", "ΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ", "αβγδϵζηθικλμνξοπρστυϕχψω"));
+            alphabets.Add(new Alphabet("ga", "ABCDEFGHILMNOPRSTUÁÉÍÓÚ", "abcdefghilmnoprstuáéíóú"));
+            alphabets.Add(new Alphabet("ga-IE", "ABCDEFGHILMNOPRSTUÁÉÍÓÚ", "abcdefghilmnoprstuáéíóú"));
+
+            if (Language != "tr" && Language != "el" && Language != "ga" && Language != "ga-IE")
+                return GetToLower();
+            else
+            {
+                var lowerWord = new StringBuilder();
+                var str = alphabets.Find(r => r.Language == Language);
```
Comment (pnklshh) [inline S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:41]: This variable could be named something meaningful

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            char getLowerChar;
+            if (positonOfChar >= 0)
+                getLowerChar = char.Parse(s: alphabets.Lower.Substring(positonOfChar, 1));
+            else
+                getLowerChar = charOfWord;
+
+            return getLowerChar;
+        }
+    }
+}
```
Comment (pnklshh) [inline S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:76]: Is this function really needed ? Could have directly used inbuilt ToLower() method provided by C#

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        public UpperToLower(string word, string language)
+        {
+            if (string.IsNullOrEmpty(language))
+            {
+                throw new ArgumentException($"'{nameof(language)}' cannot be null or empty.", nameof(language));
+            }
+            Word = word ?? throw new ArgumentNullException(nameof(word));
+            Language = language;
+
+        }
```
Comment (pnklshh) [inline S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:18]: This is good, having a class constructor and handling exceptions !

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs (hunk 1 above)
Comment (cihanozluk) [reply S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:51]: I have  just realized. thank you

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs (hunk 2 above)
Comment (cihanozluk) [reply S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:55]: last condition checking second char of the word is Uppercase

**Review by pnklshh — COMMENTED** [review]
It would be better if you can add some comments in the code.

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs (hunk 3 above)
Comment (cihanozluk) [reply S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:41]: you mean str rhigt

File: S21/cihanozluk/ToLower/ToLower/UpperToLower.cs (hunk 4 above)
Comment (cihanozluk) [reply S21/cihanozluk/ToLower/ToLower/UpperToLower.cs:76]: I wanted to create my function inste of using C# ToLower()

**Pull request closed without merging**